.pipelines/trigger.yaml

```diff
@@ -0,0 +1,51 @@
+pr:
+  branches:
+    include:
+      - master
+      - release/*
+  paths:
+    exclude:
+      - ".devcontainer"
+      - ".hooks"
+      - ".vscode"
+      - ".github"
+      - docs
+
+
+trigger:
+  branches:
+    include:
+      - gh-readonly-queue/master/*
+  tags:
+    include:
+      - "*"
+
+
+resources:
+  repositories:
+  - repository: azure-container-networking
+    type: github
+    name: Azure/azure-container-networking
+    endpoint: 'Azure-ACN RO Service Connection'
+    ref: refs/heads/feature/ob-onboard-0
+
+
+variables:
+  REPO_REF: $[ resources.repositories['azure-container-networking'].ref ]
+  REPO_COMMIT: $[ resources.repositories['azure-container-networking'].version ]
+  REPO_NAME: $[ resources.repositories['azure-container-networking'].name ]
+  REPO_TYPE: $[ resources.repositories['azure-container-networking'].type ]
+  CHANGESET_COMMIT: $[ resources.repositories['self'].version ]
+ 
+
+pool:
+  vmImage: ubuntu-latest
+
+
+extends:
+  template: /.pipelines/template.trigger.jobs.yaml@azure-container-networking
+  parameters:
+    mainRepoRef: $(REPO_REF)
+    mainRepoCommit: $(REPO_COMMIT)
+    mainRepoName: $(REPO_NAME)
+    mainRepoType: $(REPO_TYPE)
```

Comment:
Doesn't referencing them this way change it from compile time to run time? I feel like that could have negative impacts later on in development of this feature.

Reply:
No. I don't think so. I think this is different because it is using a variable from a resource type, which can guarantee that a variable exists or not.

Reply:
https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/resources-repositories-repository?view=azure-pipelines#variables

Reply:
> In each run, the metadata for a repository resource is available to all jobs in the form of runtime variables.

Ah so it is already a runtime variable.